pkg/liqonet/iptables/iptables.go

```diff
@@ -481,6 +504,25 @@ func getPreRoutingRules(tep *netv1alpha1.TunnelEndpoint) ([]IPTableRule, error)
 	return rules, nil
 }
 
+func getPreRoutingRulesPerNm(nm *netv1alpha1.NatMapping) ([]IPTableRule, error) {
+	// Check tep fields
+	if nm.Spec.ClusterID == "" {
+		return nil, &errors.WrongParameter{
+			Parameter: consts.ClusterIDLabelName,
+			Reason:    errors.StringNotEmpty,
+		}
+	}
+
+	rules := make([]IPTableRule, 0)
```

Comment:
Since the IP addresses for a remote address could be many, i would suggest to allocate the slice of the right size. You can use the length of `nm.Spec.ClusterMappings`.